corehq/apps/userreports/reports/specs.py

```diff
@@ -375,9 +378,46 @@ def restricted_to_static(cls):
         # so this column type is only available for static reports.  To release this,
         # we should require that conditions be expressed using a PreFilterValue type
         # syntax, as attempted in commit 02833e28b7aaf5e0a71741244841ad9910ffb1e5
-        return True
+        return False     # TODO True
```

Comment:
Is it expected that this will move back to `True`?

Reply:
yes, fixed